solutions/project/project-sandbox/network/nat.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Cloud NAT northamerica-northeast1
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeRouterNAT
+metadata:
+  name: project-id-nane1-nat # kpt-set: ${project-id}-nane1-nat
+  namespace: networking # kpt-set: ${namespace}
```

Comment:
is `# kpt-set: ${namespace}` needed? it doesn't seem to be part of `setters.yaml`

same thing for other resources in this file

Reply:
I will fix it

Reply:
removed with last commit